cpp/log/cert.cc

```diff
@@ -1370,12 +1504,31 @@ util::Status CertChain::IsValidSignatureChain() const {
                         "certificate chain is not loaded");
   }
 
+  // Search up the chain from the leaf for a certificate with a DSA
```

Comment:
I removed all this code for the moment. We didn't have anything to test with as we couldn't find any examples.